webapp/src/main/java/com/box/l10n/mojito/service/thirdparty/ThirdPartyTMSSmartling.java

```diff
@@ -927,14 +930,15 @@ private Long singularCount(
       String skipAssetsWithPathPattern) {
     return textUnitSearcher
         .countTextUnitAndWordCount(
-            baseParams(
-                repositoryId,
-                localeTag,
-                skipTextUnitsWithPattern,
-                skipAssetsWithPathPattern,
-                true,
-                true,
-                null))
+            this.baseParams(
```

Comment:
Now that we have `TextUnitSearcherParameters.Builder`, could we have baseParams just have the common conditions preconfigured (rootlocale =false, used filter = Used, etc.) and have it return a builder - and everything that's different from call to call to use the native builder pattern?

It should also make calls like this a lot more obvious as to what params they're passing, once replaced with the builder:
```
this.baseParams(
                    repositoryId,
                    localeTag,
                    skipTextUnitsWithPattern,
                    skipAssetsWithPathPattern,
                    false,
                    false,
                    "%")
                .build())
```
--->
```
this.baseParams()
    .repositoryIds(repositoryId)
    .pluralFormsFiltered(False)
    # ... etc
    .build())
```

Reply:
Done